terraform-tests/s3_test.go

```diff
@@ -116,4 +117,23 @@ var _ = Describe("S3", Label("S3-terraform"), Ordered, func() {
 				}))
 		})
 	})
+
+	Context("explicitly denies access to HTTP requests", func() {
```

Comment:
```suggestion
	Context("setting restrict_to_tls_requests_only to true", func() {
```

I believe this `Context` description together with the suggestion below to change the `It` description better explains what the test is doing and what it can guarantee.
This test doesn't guarantee that HTTP requests are denied, it can only guarantee that the policy has been created.

Reply:
Done!!